client/src/components/EventList/EventCard/EventCard.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import {
+  Card,
+  CardImg,
+  CardText,
+  CardBody,
+  CardTitle,
+  CardSubtitle,
+  Button,
+} from 'reactstrap';
+import { Link } from 'react-router-dom';
+
+export class EventCard extends Component {
```

Comment:
@QaseemLodhi @taleymirza please use the function syntax when no state is required

i.e. EventCard = (props) => { ...